plugins/quickinspector/quickitemgeometry.cpp

```diff
@@ -80,6 +86,22 @@ void QuickItemGeometry::initFrom(QQuickItem *item)
     }
     x = item->x();
     y = item->y();
+
+    const QMetaObject *mo = item->metaObject();
+    if (mo->property(mo->indexOfProperty("padding")).isValid()) {
```

Comment:
Others like background and contentItem are checked from line 57-62.
Do I missed some other properties ?